ex1.py

```diff
@@ -0,0 +1,18 @@
+
+try:
+    def calculate_average ():
+        counter = 0
+        sum = 0
+        function_stop = "q"
+        num = input("enter num if you want to stop enter " + function_stop + " ")
+        while num != function_stop :
+            sum += float(num)
+            counter += 1
+            num = input("enter num ")
```

Comment:
try and except should be on this line

Reply:
if there an except so he stays in the except and not going back to try